core/main/src/processor/main_context_processor.rs

```diff
@@ -145,110 +111,16 @@ impl MainContextProcessor {
-                .configuration
-                .features
-                .privacy_settings_storage_type
-                == PrivacySettingsStorageType::Sync
-            {
-                debug!("Privacy settings storage type is set as sync so starting cloud monitor");
-                if let Some(account_session) = state.session_state.get_account_session() {
-                    debug!("Successfully got account session");
-                    if !update_token {
-                        let sync_response = state
-                            .get_client()
-                            .send_extn_request(SyncAndMonitorRequest::SyncAndMonitor(
-                                SyncAndMonitorModule::Privacy,
-                                account_session.clone(),
-                            ))
-                            .await;
-                        debug!("Received Sync response for privacy: {:?}", sync_response);
-                        let sync_response = state
-                            .get_client()
-                            .send_extn_request(SyncAndMonitorRequest::SyncAndMonitor(
-                                SyncAndMonitorModule::UserGrants,
-                                account_session.clone(),
-                            ))
-                            .await;
-                        debug!(
-                            "Received Sync response for user grants: {:?}",
-                            sync_response
-                        );
-                    } else {
-                        debug!("cap already available so just updating the token alone");
-                        let update_token_response = state
-                            .get_client()
-                            .send_extn_request(SyncAndMonitorRequest::UpdateDistributorToken(
-                                account_session.token.clone(),
-                            ))
-                            .await;
-                        debug!("Cap token:account is already in available state. just updating Token res: {:?}", update_token_response);
-                    }
-                    //sync up partner exclusion data and setup polling thread for refreshing it.
-                    Self::sync_partner_exclusions(state).await;
-                }
-            }
         }
     }
 
-    fn handle_internet_connection_change(
-        state: &PlatformState,
-        internet_state: &Option<InternetConnectionStatus>,
-        // internet_state: &InternetConnectionStatus,
-    ) {
-        debug!("handling internet connection change: {:?}", internet_state);
-        let internet_state = match internet_state {
-            Some(state) => state,
-            None => return,
-        };
-
-        if matches!(internet_state, InternetConnectionStatus::FullyConnected) {
-            // get internet monitoring interval from platformState configuration
-            let interval_in_seconds = state
-                .get_device_manifest()
```

Comment:
A few questions:

1. If we're no longer maintaining the state of the internet connection, we're ok with waiting for the connection timeout however long that is? I thought we added state monitoring specifically to avoid that so wanted to make sure we're on the same page.
2. Did we also remove the monitoring interval and related fields from the device manifest?